### PR TITLE
Qualcomm_11-6-2019_2

### PR DESCRIPTION
**Pull request by QC-CNA**

### CODE REVIEW

File: 2019/10xxx/CVE-2018-13916.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{
```
Comment (cve-team) [inline 2019/10xxx/CVE-2018-13916.json]: Expected location (2018/13xxx/CVE-2018-13916.json) of file does not match actual location (2019/10xxx/CVE-2018-13916.json)

Comment (QC-CNA) [reply 2019/10xxx/CVE-2018-13916.json]: please let me know how i can make changes, can i delete CVE-2018-13916 from this checkin and add it later

**Comment by QC-CNA** [comment]
Removed CVE-2018-13916 from 2019 folder and updated Qualcomm_11-6-2019_2 branch. Please merge

Thanks
Sameesh

**Comment by ccoffin** [comment]
Sameesh,

Sorry for the delay in getting back to you on this request. There is one more issue that appears to affect all 25 files. In each case it looks like the reference is not pointing directly to the advisory that contains the CVE, but pointing to a higher-level advisory view instead.

CVE-2018-13916 for example points to https://www.qualcomm.com/company/product-security/bulletins, but should probably point to https://www.qualcomm.com/company/product-security/bulletins/october-2019-bulletin instead as this advisory discusses the CVE specifically. 

Please update the references in each case to provide the direct advisory reference. 

For more information on the requirements for a CNA submission, see section 2.1.1 of the CNA Rules document at https://cve.mitre.org/cve/cna/rules.html#Section_2_1_assignment_rules.

Feel free to reach out if you have any questions.

Regards,

Chris

**Comment by QC-CNA** [comment]
Hi Chris,

I have updated Qualcomm_11-6-2019_2 branch with changes to all 25 files as per your suggestion. Please review and approve

Thanks
Sameesh

**Pull request merged**